rustls/src/error.rs

```diff
@@ -5,6 +5,7 @@ use core::fmt;
 use pki_types::{ServerName, UnixTime};
 #[cfg(feature = "std")]
 use std::time::SystemTimeError;
+use webpki::KeyUsage;
```

Comment:
I think this file has a small pre-existing import order style issue. Want to fix it up now?

Both the `pki_types` and `webpki` imports should be separate from the `alloc/core/std` imports above.

Reply:
Yup, fixed!